src/components/visualizations/typings/chart.ts

```diff
@@ -0,0 +1,15 @@
+// (C) 2020 GoodData Corporation
+
+import { Execution } from "@gooddata/typings";
+
+type UnwrappedAttributeHeader = Execution.IAttributeHeader["attributeHeader"];
+
+export interface IUnwrappedAttributeHeaderWithItems extends UnwrappedAttributeHeader {
+    items: Execution.IResultAttributeHeaderItem[];
+}
+type UnwrappedAttributeHeaderItem = Execution.IResultAttributeHeaderItem["attributeHeaderItem"];
+export interface IAttributeItem extends UnwrappedAttributeHeaderItem {
+    attribute: IUnwrappedAttributeHeaderWithItems;
+}
+
+export type UnwrappedMeasureHeaderItem = Execution.IMeasureHeaderItem["measureHeaderItem"];
```

Comment:
It could be re-arranged
```

import { Execution } from "@gooddata/typings";

type UnwrappedAttributeHeader = Execution.IAttributeHeader["attributeHeader"];
type UnwrappedAttributeHeaderItem = Execution.IResultAttributeHeaderItem["attributeHeaderItem"];

export type UnwrappedMeasureHeaderItem = Execution.IMeasureHeaderItem["measureHeaderItem"];

export interface IUnwrappedAttributeHeaderWithItems extends UnwrappedAttributeHeader {
    items: Execution.IResultAttributeHeaderItem[];
}

export interface IAttributeItem extends UnwrappedAttributeHeaderItem {
    attribute: IUnwrappedAttributeHeaderWithItems;
}

```

Reply:
fixed